python/paddle/fluid/trainer.py

```diff
@@ -348,6 +423,41 @@ def _get_or_create_parallel_executor(self):
                 loss_name=self.train_func_outputs[0].name)
         return self._get_parallel_executor()
 
+    def _clean_checkpoint(self):
+        if not self.checkpoint:
```

Comment:
use `assert` instead of return directly, otherwise we don't know whether this function success.